### PR TITLE
Added 4.3.3 Release notes.

### PR DESCRIPTION
**Pull request by sagarkapare**

### CODE REVIEW

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 1]
```diff
@@ -31,6 +31,48 @@ Cask Data Application Platform Release Notes
...
+Bug Fixes
+---------
+
+- :cask-issue:`CDAP-7052` - Fixed a bug that prevented MapReduce AM logs from YARN to show the right URI.
+
+- :cask-issue:`CDAP-7644` - Added CLI command to fetch service logs.
+
+- :cask-issue:`CDAP-12774` - Increased the dataset changeset size and limit to integer max by default.
+
+- :cask-issue:`CDAP-12900` - Fixed bug where macro for output schema of a node was not saved when the user closed the node properties modal.
```
Comment (prinam) [inline cdap-docs/reference-manual/source/release-notes.rst]: "Fixed a bug where..."

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 2]
```diff
@@ -31,6 +31,48 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-12930` - Fixed a bug where explore queries would fail against paths in hdfs encryption zones, for certain hadoop distributions.
```
Comment (prinam) [inline cdap-docs/reference-manual/source/release-notes.rst]: "HDFS encryption zones", "Hadoop"

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 3]
```diff
@@ -31,6 +31,48 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-12945` - Fixed bug where old connection was not removed from the pipeline config when we moved that connection's pointer to another node.
```
Comment (prinam) [inline cdap-docs/reference-manual/source/release-notes.rst]: "Fixed a bug where the old connection is not removed from the pipeline config when you move the connection's pointer to another node."

File: cdap-docs/reference-manual/source/release-notes.rst [hunk 4]
```diff
@@ -31,6 +31,48 @@ Cask Data Application Platform Release Notes
...
+
+- :cask-issue:`CDAP-12946` - Fixed a bug in the pipeline planner where pipelines that used an action before multiple sources would either fail to deploy or deploy with an incorrect plan.
+
+- :cask-issue:`CDAP-12970` - Fixed a dependency bug that could cause HBase region servers to deadlock during a cold start.
+
+- :cask-issue:`CDAP-13002` - Fixes an issue with the retrieval of non-ASCII strings from Table datasets.
```
Comment (anwar6953) [inline cdap-docs/reference-manual/source/release-notes.rst]: nit - `Fixes` --> `Fixed`, just to be consistent with other release notes

File: cdap-docs/reference-manual/source/release-notes.rst (hunk 1 above)
Comment (sagarkapare) [reply cdap-docs/reference-manual/source/release-notes.rst]: fixed.

File: cdap-docs/reference-manual/source/release-notes.rst (hunk 2 above)
Comment (sagarkapare) [reply cdap-docs/reference-manual/source/release-notes.rst]: fixed.

File: cdap-docs/reference-manual/source/release-notes.rst (hunk 3 above)
Comment (sagarkapare) [reply cdap-docs/reference-manual/source/release-notes.rst]: fixed.

File: cdap-docs/reference-manual/source/release-notes.rst (hunk 4 above)
Comment (sagarkapare) [reply cdap-docs/reference-manual/source/release-notes.rst]: fixed.

**Comment by sagarkapare** [comment]
@prinam @anwar6953 Addressed comments. please take a look again. thanks!

**Review by anwar6953 — APPROVED** [review]

**Pull request merged**